### PR TITLE
Add spark example to Sample Notebook

### PR DESCRIPTION
**Pull request by kristin-kim**
Add a simple spark example to existing sample Notebook of <Dataproc Running Notebooks> folder

### CODE REVIEW

File: examples/dataproc-running-notebooks/notebooks/sample_notebook.ipynb [hunk 1]
```diff
@@ -7,14 +7,109 @@
...
+    "# Run a sample Spark Session with a sample CSV file\n",
+    "\n",
+    "from pyspark.sql import SparkSession\n",
+    "from pyspark.sql.functions import avg, sum, max\n",
+    "spark = SparkSession \\\n",
+    "    .builder \\\n",
+    "    .appName(\"Spark Session for Bike Sharing Data\") \\\n",
+    "    .getOrCreate()\n",
+    "\n",
+    "path=\"gs://kristin_serverless_pyspark/bikesharingdemand-train.csv\"\n",
```
Comment (NiloFreitas) [inline examples/dataproc-running-notebooks/notebooks/sample_notebook.ipynb:32]: Is this csv too big? If it is small is better to have it under a resources folder and not hardcode a GCS path that may not exist in the future

File: examples/dataproc-running-notebooks/notebooks/sample_notebook.ipynb [hunk 2]
```diff
@@ -7,14 +7,109 @@
    "source": [
     "# Sample Notebook\n",
     "The output of this notebook will be executed via PySpark job then will be stored in the specified notebook path in notebook_args in the Composer DAGs. \n",
+    "\n",
     "This notebook let users \n",
+    "* run a sample Spark Session with a sample CSV file\n",
     "* verify if GCS buckets are properly mounted as a file system and\n",
     "* execute Python files that are stored in mounted GCS buckets by !python and %run commands and\n"
```
Comment (NiloFreitas) [inline examples/dataproc-running-notebooks/notebooks/sample_notebook.ipynb:14]: Do we need to keep this part of the implementation? I mean the gcsfuse and running a python file from the notebook. I thought that you were going to create a separate example to demonstrate that, and keep this project running only PySpark

**Review by NiloFreitas — COMMENTED** [review]
Hi Kristin. Please take a look at my comments. Cheers

File: examples/dataproc-running-notebooks/notebooks/sample_notebook.ipynb (hunk 2 above)
Comment (agold-rh) [reply examples/dataproc-running-notebooks/notebooks/sample_notebook.ipynb:14]: @kristin-kim If you can address the review questions, we can try to get this merged.

**Comment by agold-rh** [comment]
Closed as stale. Please re-open if I'm wrong.

**Pull request closed without merging**